src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -88,7 +89,14 @@ protected function withoutEvents()
             $this->firedEvents[] = $called;
         });
 
+        $mock->shouldReceive('until')->andReturnUsing(function ($called) {
+            $this->firedEvents[] = $called;
+
+            return true;
+        });
+
         $this->app->instance('events', $mock);
+        Model::setEventDispatcher($mock);
```

Comment:
My point was that as long as the capsule manager calls `bootEloquent` during initialization (https://github.com/laravel/framework/blob/5.2/src/Illuminate/Database/Capsule/Manager.php#L132) the mock will be replaced. That being said explicitly resetting it after initialization couldn't hurt. I've just pushed a commit to do this.